lib/index.js

```diff
@@ -189,6 +188,27 @@ module.exports.render = function(options, cb) {
     };
   }
 
+  if (options.functions) {
+    Object.keys(options.functions).forEach(function(signature) {
+      var callback = options.functions[signature];
+
+      options.functions[signature] = function() {
+        var args = Array.prototype.slice.call(arguments),
+            bridge = args.pop();
+
+        function done(data) {
+          bridge.success(data);
+        }
+
+        var result = callback.apply(this, args.concat(done));
```

Comment:
@Jakobo, wow image manipulation, like generating and mapping sprites? One day I will learn all the powers and mysteries behind `this.conext`, but I don't get time to actually *use* node-sass like others do! :smile_cat: